src/transformers/feature_extraction_utils.py

```diff
@@ -286,22 +286,24 @@ def from_pretrained(
 
         return cls.from_dict(feature_extractor_dict, **kwargs)
 
-    def save_pretrained(self, save_directory: Union[str, os.PathLike]):
+    def save_pretrained(self, save_directory: Union[str, os.PathLike], processor_class_str: Optional[str] = None):
```

Comment:
this allows us to save the _processor_class_ in the feature_extractor config so that:

```python
auto_processor.save_pretrained("./")
AutoProcessor.from_pretrained("./")
```

is always possible for Wav2Vec2